docker-compose.test.yml

```diff
@@ -1,5 +1,3 @@
-version: "3.3"
```

Comment:
Removed because deprecated, and docker compose in ubuntu 24.04 warns against using it.